src/current/v23.2/common-issues-to-monitor.md

```diff
@@ -106,6 +110,12 @@ Issues at the storage layer, including an [inverted LSM]({% link {{ page.version
 {% include {{ page.version.version }}/prod-deployment/resolution-inverted-lsm.md %}
 {{site.data.alerts.end}}
 
+{{site.data.alerts.callout_success}}
+[XXX](): <font color="red">QUESTION</font>: the instructions above say to reduce your workload concurrency, so it seems like maybe we should NOT be recommending increasing compaction concurrency here? b/c if we had IOPs and CPU headroom we would not be in this bad state, and thus increasing compaction concurrency would use *even more* IO / CPU right?  That's my sense anyway but please confirm.
```

Comment:
@jbowens depending on your thoughts in the other convo we could remove this blurb here too for now.  Although this section does seem more "purely about storage", it seems to share the property that maybe one's first try setting this env var shouldn't be during an escalation / overload situation?